app/views/organizations/_sidebar_additional.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="sidebar-bg" id="sidebar-bg-right"></div>
   <div class="side-bar sidebar-additional showing" id="sidebar-additional">
     <% @organization.sponsorships.live.find_each do |sponsorship| %>
-      <div class="sidebar-sponsorship-level" style="background:<%= sponsorship.decorate.level_background_color %>">
```

Comment:
Check with @benhalpern about this. They probably change this in the database when changing sponsors.

Reply:
Yeah, but if we lose this from the design it's fine with me.

Reply:
Good call though